packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -23,7 +23,7 @@ import {StateChange, Logout} from './services/agent/types';
 import {getErrorDetails} from './services/core/Utils';
 import {Profile, WelcomeEvent} from './services/config/types';
 import {AGENT_STATE_AVAILABLE} from './services/config/constants';
-import {ConnectionLostDetails} from './services/core/websocket/types';
+import {ConnectionLostDetails} from './services/core/WebSocket/types';
```

Comment:
For some reason, this was not letting me build and coming as an error, so I added this change. Will update once Priya's PR goes in.